ProcessMaker/Http/Controllers/Api/TaskController.php

```diff
@@ -269,8 +278,11 @@ public function update(Request $request, ProcessRequestToken $task)
     private function handleQueryException($e)
     {
         $regex = '~Column not found: 1054 Unknown column \'(.*?)\' in \'where clause\'~';
+
         preg_match($regex, $e->getMessage(), $m);
 
+        dump($m);
```

Comment:
@runyan-co Is this meant to remain here?

Reply:
Whoops, haha, I just pushed a commit to remove it. 👍🏻